routes.js

```diff
@@ -61,6 +60,12 @@ import {
 } from 'issue';
 
 const sharedRoutes = {
+  OrgRepositoryList: {
```

Comment:
Perhaps it's better to use the full name here (`OrganizationRepositoryList`), because the other screens start with the `Organization`, not the `Org` (to support consistency)

Reply:
Absolutely